### DISCUSSION TITLE
The toolbar icons look bad in home tab

### DISCUSSION THREAD

**Discussion by ne0-wu** [sumatrapdfreader/sumatrapdf#3301] [Ideas]
Aliasing artifacts occur on the toolbar icons when in the home tab. Is it possible to replace them?

![image](https://user-images.githubusercontent.com/26689838/221104920-e465a16c-a52a-438e-8301-85e2cb1ff9ac.png)

**Comment by kjk** [comment]
It's how toolbar control does "disabled" look by graying them out.

Not much I can do about it except rewrite the whole toolbar to not use toolbar control.

**Comment by ne0-wu** [comment]
Maybe it would be cool to just hide the toolbar in home tab? I think this would be easy to implement. Most of them are disabled in this condition anyway.

**Reply by GitHubRulesOK** [reply]
You can switch them off or use bigger size

![image](https://user-images.githubusercontent.com/12754913/227183385-8faa2cff-6271-4068-a55d-4ea76e36dc04.png)
